### DISCUSSION TITLE
plugin

### DISCUSSION THREAD

**Discussion by olatoyemmanuel** [Cyfrin/foundry-full-course-cu#2320] [Q&A]
please i was unable to see zksync when i want to connect to plugin, what can i do? please

**Comment by EngrPips** [comment]
Hello @olatoyemmanuel, You should refresh your browser and then retry. Let's know if you refreshing the browser help resolve the issue.

**Reply by olatoyemmanuel** [reply]
thanks @EngrPips for your reply

but the suggestion didn't work

the problem i am facing is, any time i sreach for zksync in the plugin, i can't seen to find it

**Reply by EngrPips** [reply]
Hello @olatoyemmanuel, could you share your screen when you search so that we can see how you are doing the search.

**Comment by olatoyemmanuel** [comment]
![WhatsApp Image 2024-08-30 at 4 33 52 AM](https://github.com/user-attachments/assets/5ca9bd36-d58a-4668-85a7-0dca27ff6074)

**Comment by EngrPips** [comment]
<img width="1440" alt="Screenshot 2024-08-30 at 07 55 58" src="https://github.com/user-attachments/assets/0e54cecf-1213-4678-97f5-e96787202e39">

It seems to work for me, and I'm not sure why it isn't working for you. Can you try another browser?

**Comment by olatoyemmanuel** [comment]
okay
